pkgs/development/tools/mani/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildGoModule, fetchFromGitHub, lib, git}:
+
+buildGoModule rec {
+  pname = "mani";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "alajmo";
+    repo = "mani";
+    rev = "v${version}";
+    sha256 = "sha256-9rcgPeYFHdIN73K0zGPEHqFFLFkVYkNYRXJ+0/Zo4zI=";
+  };
+
+  vendorSha256 = "sha256-ZivzDfjx2djzS0Xm3GISK3zpB5fUUMgy2o4Ti1Z9wMM=";
+
+  buildInputs = [ git ];
+
+  postInstall = ''
+    # generate completion scripts for mani
+    export HOME="$NIX_BUILD_TOP"
+    mkdir -p $out/share/{bash-completion/completions,fish/vendor_completions.d,zsh/site-functions}
+    $out/bin/mani completion bash > "$out/share/bash-completion/completions/mani"
+    $out/bin/mani completion fish > $out/share/fish/vendor_completions.d/mani.fish
+    $out/bin/mani completion zsh > $out/share/zsh/site-functions/_mani
```

Comment:
Though that works, the standard  nixpkgs way uses `installShellCompletion`.

Reply:
I figured out why I couldn't get it working before. Updated.